app.py

```diff
@@ -377,9 +378,8 @@ def main():
 
     path_git = path_to_git()
     path_pic = path_pic = pjoin(path_git, "git-hub/PRs.png")
-    if os.path.exists(path_pic):
-        return path_pic
-    return None
+    if not os.path.exists(path_pic):
```

Comment:
The problem I have with this approach in general is that we can only have *one* plot.  But we will certainly have many more coming soon.